docs/excel/excel-add-ins-workbooks.md

```diff
@@ -258,6 +258,31 @@ The Excel API also lets add-ins turn off calculations until `RequestContext.sync
 context.application.suspendApiCalculationUntilNextSync();
 ```
 
+## Save the workbook
+
+> [!NOTE]
+> The `Workbook.save(saveBehavior)` function is currently available only in public preview. [!INCLUDE [Information about using preview APIs](../includes/using-preview-apis.md)]
+
+The workbook is saved to persistent storage with `Workbook.save(saveBehavior)`. The `save` method takes a single, optional `SaveBehavior` enum as a parameter. This specifies whether the user is prompted about the file name and save location. `save` saves the workbook without ever prompting the user. If the workbook has never been saved, then the file is automatically saved to the default location. `save` is the default behavior. `prompt` gives the user the Excel save prompt the first time the workbook is saved. It does not prompt the user if the workbook has been previously saved.
+
+> [!CAUTION]
+> If the user is prompted to save and cancels the operation, `save` throws an exception.
+
+```js
+context.workbook.save(Excel.SaveBehavior.prompt);
+```
+
+## Close the workbook
+
+> [!NOTE]
+> The `Workbook.close(closeBehavior)` function is currently available only in public preview. [!INCLUDE [Information about using preview APIs](../includes/using-preview-apis.md)]
+
+`Workbook.close(closeBehavior)` closes the workbook (and your add-in associated with the workbook). The Excel application remains open. The `close` method takes a single, optional `CloseBehavior` enum as a parameter. This specifies whether the workbook is saved before closing. `save` saves the workbook; prompting the user if the workbook has never previously been saved. `save` is the default behavior. `skipSave` immediately closes the workbook without saving, potentially losing information.
```

Comment:
Suggest that you rework this paragraph for clarity,...something structured more like this (assuming my interpretation is accurate, of course):

Your add-in can close the current workbook (along with your add-in that's associated with the workbook) by using `Workbook.close(closeBehavior)`. The Excel application remains open after `Workbook.close(closeBehavior)` is called. The `close` method takes a single, optional parameter that can one of the following two values:
- `Excel.CloseBehavior.save`: The file is saved before closing; if the file has not been saved previously, the user will be prompted to specify file name and save location.
- `Excel.CloseBehavior.skipSave`: The file is immediately closed, without saving. Any unsaved changes will be lost.

If no input parameter is specified, behavior of the `Workbook.close` method is the same as if `Excel.CloseBehavior.save` had been specified.